Library/Homebrew/test/rubocops/formula_desc_cop_spec.rb

```diff
@@ -91,6 +91,16 @@ class Foo < Formula
       RUBY
     end
 
+    it "When the description ends with a full stop" do
+      expect_offense(<<~RUBY, "/homebrew-core/Formula/foo.rb")
+        class Foo < Formula
+          url 'http://example.com/foo-1.0.tgz'
+          desc 'Description with a full stop at the end.'
+                                                       ^ Description shouldn\'t end with a full stop
```

Comment:
Are you sure you need to escape the `\'`? If so: no worries.

Reply:
The other occurrences of "shouldn't" in this file escape the apostrophe.